### PR TITLE
Convert service account into app

### PR DESCRIPTION
**Pull request by korycins**
- Create `App` module
- Move (to `App` module) and Rename(to `App`) `ServiceAccount` model to separate module
- Deprecate `ServiceAccount` mutations/queries
- Handle ServiceAccount queries/mutations with `App` logic
- Add `App` queries/mutations based on `ServiceAccount`

### CODE REVIEW

File: saleor/graphql/account/schema.py [hunk 1]
```diff
@@ -296,25 +290,50 @@ class AccountMutations(graphene.ObjectType):
...
+    service_account_update = ServiceAccountUpdate.Field(
+        deprecation_reason=(
+            "Use the `appUpdate` mutation instead. This field will be removed after "
+            "2020-07-31."
+        )
+    )
+    service_account_delete = ServiceAccountDelete.Field(
+        deprecation_reason=(
+            "Use the `appDelete` mutation instead. This field will be removed after "
+            "2020-07-31."
```
Comment (NyanKiyoshi) [inline saleor/graphql/account/schema.py:308]: Souldn't we use a release number instead?

File: saleor/graphql/webhook/mutations.py [hunk 2]
```diff
@@ -132,6 +139,16 @@ class Meta:
         error_type_class = WebhookError
         error_type_field = "webhook_errors"
 
+    @classmethod
+    def clean_input(cls, info, instance, data, input_cls=None):
+        cleaned_input = super().clean_input(info, instance, data, input_cls)
+        # This should be removed after we drop service_account
```
Comment (NyanKiyoshi) [inline saleor/graphql/webhook/mutations.py:145]: Please open an issue to track this

Comment (korycins) [reply saleor/graphql/webhook/mutations.py:145]: I already did that https://github.com/mirumee/saleor/issues/5521 ;)

File: saleor/graphql/account/schema.py (hunk 1 above)
Comment (korycins) [reply saleor/graphql/account/schema.py:308]: We've changed approach to use date instead of version

**Pull request merged**